src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -363,6 +363,9 @@
 
   "skyblocker.config.helpers.enableNewYearCakesHelper": "Enable New Year Cakes Helper",
   "skyblocker.config.helpers.enableNewYearCakesHelper.@Tooltip": "Highlights the missing new year cakes green and the cakes you have already red.\n\nRequires you to open your cake bag at least once to work.",
+  "skyblocker.config.helpers.enableBitsHelper": "Highlight best Bits Shop offer",
+  "skyblocker.config.helpers.enableBitsHelper.@Tooltip": "Highlights best offers in Bits Shop (Community Store). Also shows coins per bit. Green if best offer that is in list of items that quickly sell. Yellow for best offer overall - may include stuff that sells once in a year. \n\nBoth may include price manipulated items - please use common sense or at least 1day/3day avg tooltip",
```

Comment:
- Highlights best offers in __the__ Bits Shop

The place is actually called the Community Shop too

Reply:
Pretty sure most players call it "Bits Shop" but ye ima change wording